src/ansys/dpf/core/data_sources.py

```diff
@@ -119,14 +134,16 @@ def __init__(self, result_path=None, data_sources=None, server=None):
         if result_path is not None:
             self.set_result_file_path(result_path)
 
-    def set_result_file_path(self, filepath, key=""):
-        """Add a result file path to the data sources.
+    def set_result_file_path(
+        self, filepath: Union[str, os.PathLike], key: Optional[str] = ""
+    ) -> None:
+        """et the main result file path to the data sources.
```

Comment:
@luisaFelixSalles I think there is a typo here

Reply:
```suggestion
        """Set the main result file path to the data sources.
```